docs/config.md

```diff
@@ -55,6 +55,35 @@ for more details.
 | `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | * | source |
 | `NetRuntime` | [OpenTelemetry.Instrumentation.Runtime](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime) | * | source |
 
+## Logs
+
+Logs from ASP.NET Core apps could be captured using two options. First option is
+to use HostingStartup and the second option is to rely on CLR Profiler
+Instrumentation. Recommended option for ASP.NET Core is to use HostingStartup.
+For other .NET Core apps logs could be collected only through .NET CLR Profiler
+Instrumentation.
+
+### Enable Logs using HostingStartup
+
+| Environment variable | Required value |
+|-|-|
+| `ASPNETCORE_HOSTINGSTARTUPASSEMBLIES` | `OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper` |
+
+### Enable Logs using .NET CLR Profiler
+
+Please follow [instructions](config.md#net-clr-profiler) here to collect logs
+using .NET CLR Profiler.
+
+### Logs Parse and Format Options
```

Comment:
This heading reads a little awkwardly for me. Maybe it should be something like the following.

```suggestion
### Parsing and Format Options for Logs
```